livekit-plugins/livekit-plugins-turn-detector/livekit/plugins/turn_detector/eou.py

```diff
@@ -13,10 +13,21 @@
 from .log import logger
 
 HG_MODEL = "livekit/turn-detector"
+ONNX_FILENAME = "model_quantized.onnx"
 PUNCS = string.punctuation.replace("'", "")
 MAX_HISTORY = 4
 
 
+def _download_from_hf_hub(repo_id, filename):
```

Comment:
shouldn't this be in download_files? we don't want to download during runtime

Reply:
should be fixed now